ghost/i18n/locales/es/portal.json

```diff
@@ -6,63 +6,63 @@
     "Back": "Atrás",
-    "Back to Log in": "Volver al inicio de sesión",
+    "Back to Log in": "Atrás para Iniciar Sesión",
     "Billing info": "Información de pago",
     "Black Friday": "Black Friday",
-    "Cancel anytime.": "Cancelación en cualquier momento.",
+    "Cancel anytime.": "Cancela en cualquier momento.",
     "Cancel subscription": "Cancelar suscripción",
     "Cancellation reason": "Razón de cancelación",
     "Change": "Cambiar",
     "Check spam & promotions folders": "Comprueba la carpeta de spam y promociones",
-    "Check with your mail provider": "Comprueba con tu proveedor de correo",
+    "Check with your mail provider": "Comprueba con tu proveedor de correo electrónico",
     "Choose": "Elige",
     "Choose a different plan": "Elige un plan diferente",
-    "Choose your newsletters": "Elige tus boletines",
+    "Choose your newsletters": "Elige tus boletínes",
     "Click here to retry": "Haz click aquí para reintentar",
     "Close": "Cerrar",
     "Comments": "Comentarios",
     "Complimentary": "Complementario",
     "Confirm": "Confirmar",
     "Confirm cancellation": "Confirmar cancelación",
     "Confirm subscription": "Confirmar suscripción",
-    "Contact support": "Contacta a soporte técnico",
+    "Contact support": "Contactar a soporte técnico",
     "Continue": "Continuar",
     "Continue subscription": "Continuar suscripción",
-    "Could not sign in. Login link expired.": "No se pudo iniciar sesión. El enlace de inicio ha expirado.",
+    "Could not sign in. Login link expired.": "No se pudo iniciar sesión. El enlace de inicio de sesión ha expirado.",
     "Could not update email! Invalid link.": "¡No se pudo actualizar el correo electrónico! Enlace inválido.",
     "Create a new contact": "Crea un nuevo contacto",
     "Current plan": "Plan actual",
     "Delete account": "Eliminar cuenta",
-    "Didn't mean to do this? Manage your preferences <button>here</button>.": "¿No quisiste hacer esto? Puedes gestionar tus preferencias <button>acá</button>.",
+    "Didn't mean to do this? Manage your preferences <button>here</button>.": "¿No quisiste hacer esto? Puedes gestionar tus preferencias <button>aquí</button>.",
     "Don't have an account?": "¿No tienes una cuenta?",
     "Edit": "Editar",
     "Email": "Correo electrónico",
     "Email newsletter": "Boletín informativo por correo electrónico",
-    "Email preferences": "Preferencias",
+    "Email preferences": "Preferencias de correo electrónico",
     "Emails": "Correos electrónicos",
     "Emails disabled": "Correos electrónicos desactivados",
-    "Ends {{offerEndDate}}": "Termina el {{offerEndDate}}",
+    "Ends {{offerEndDate}}": "Termina él {{offerEndDate}}",
     "Error": "Error",
-    "Expires {{expiryDate}}": "Expira el {{expiryDate}}",
-    "Forever": "para siempre",
+    "Expires {{expiryDate}}": "Expira él {{expiryDate}}",
```

Comment:
Is this (él) correct?

Reply:
Just rechecked, El is indeed without accent. Sorry about that, consulted pronouns and the actual dictionary and definitions on the RAE dictionary. "Expira el {{expiryDate}}" like it is written.

Reply:
Those are the only two that are indeed wrong, the rest is fine.

Reply:
Thanks for confirming. If you could please update the PR, I'll be glad to mark it approved with your fixes.

Reply:
Sure,

Reply:
[portal.json](https://github.com/user-attachments/files/17136930/portal.json)

That's the file changed.

Reply:
OK, so what you need to do is go back into the editor on this branch (fochoaog:patch-2), make the edits (follow the tutorial if needed), and then commit them.  That should automatically update the PR.